front_end/helper.py

```diff
@@ -243,3 +244,29 @@ def show_hidden(request_handler):
     if "show_hidden" not in request_handler.request.query_arguments:
         return False
     return request_handler.request.query_arguments["show_hidden"][0].decode().lower() == "true"
+
+def get_dict_of_dates():
+    date_dict = {}
+    years = []
+    months = []
+    days = []
+
+    for i in range(1, 10):
+        months.append("0" + str(i))
+    for i in range(10,13):
+        months.append(str(i))
+    for i in range(1, 10):
+        days.append("0" + str(i))
+    for i in range(10,32):
+        days.append(str(i))
+
+    dateTimeObj = datetime.now()
+    currYear = dateTimeObj.year
+    yearAbrev = int(currYear[2:])
+    for i in range(20, yearAbrev+1):
+        years.append(str(i))
+
+    date_dict[0] = years
```

Comment:
I would prefer that the keys to this dictionary be strings (for example, "days", "months", and "years") rather than integers. Then you don't have to remember (in the code that invokes this function) what 0, 1, and 2 stand for. But maybe even better would be to just return years, months, and days from this function rather than putting them in a dictionary. In Python, you can return multiple values from a function.